### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carlbomsdata**
Added more details of how to setup development environment for Jekyll.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,29 @@ This repository contains the source of the https://docs.fyne.io website.
...
+```
+
+### Install dependencies
+```bash
+bundle install
+```
+
+### Start the local webserver
+```bash
+bundle exec jekyll serve --host 0.0.0.0
```
Comment (andydotxyz) [inline README.md:32]: Is the host required? it looks a little confusing for first timers.

File: README.md [hunk 2]
```diff
@@ -7,9 +7,29 @@ This repository contains the source of the https://docs.fyne.io website.
...
 gem install bundler jekyll
-bundle exec jekyll serve
+```
+
+### Clone the Fyne docs repository
+```bash
+git clone https://github.com/fyne-io/docs.fyne.io.git
+```
+
+### Change directory in to the repository
```
Comment (andydotxyz) [inline README.md:20]: Does this need to be in a new section? It feels like cd after git clone could be all under the Clone header

**Review by andydotxyz — COMMENTED** [review]
Thanks so much for doing this.
Just some questions inline about if it could be simplified.

File: README.md (hunk 1 above)
Comment (carlbomsdata) [reply README.md:32]: --host is not necessary; just of ease of use. We can remove it :)

File: README.md (hunk 2 above)
Comment (carlbomsdata) [reply README.md:20]: Thats true :)

Comment (carlbomsdata) [reply README.md:20]: Sorry i am fairly new to git (still learning). Should i make a new PR?

**Comment by carlbomsdata** [comment]
Please see the updated changes :)

**Review by andydotxyz — APPROVED** [review]
Thanks. Apologies for the delay

**Pull request merged**